design/media_types.go

```diff
@@ -93,3 +93,55 @@ var WorkItemType = MediaType("application/vnd.workitemtype+json", func() {
 	})
 
 })
+
+// Tracker configuration
+var Tracker = MediaType("application/vnd.tracker+json", func() {
+	TypeName("Tracker")
+	Description("Tracker configuration")
+	Attribute("id", String, "unique id per tracker")
+	Attribute("url", String, "URL of the tracker")
+	Attribute("type", String, "Type of the tracker")
+
+	Required("id")
+	Required("url")
+	Required("type")
+
+	View("default", func() {
+		Attribute("id")
+		Attribute("url")
+		Attribute("type")
+	})
+
+	View("link", func() {
+		Attribute("id")
+	})
+})
+
+// TrackerQuery represents the search query with schedule
+var TrackerQuery = MediaType("application/vnd.trackerquery+json", func() {
+	TypeName("TrackerQuery")
+	Description("Tracker query with schedule")
+	Attribute("id", String, "unique id per installation")
+	Attribute("version", Integer, "Version for optimistic concurrency control")
```

Comment:
See https://github.com/almighty/almighty-core/issues/155

It's also unclear to me why we even need optimistic locking here.